start_vscode_sharc.sh

```diff
@@ -339,7 +339,6 @@ source \${HOME}/.bashrc
 module load $VSC_MODULE_COMMAND
 export XDG_RUNTIME_DIR="\$HOME/vsc_runtime"
 VSC_IP_REMOTE="\$(hostname)"
-#VSC_PORT_REMOTE="8899"
```

Comment:
Can we remove this edit from this PR and keep it in https://github.com/rcgsheffield/vscoderemote_sheffield_hpc/pull/2 to keep each PR contiguous with their naming?

Reply:
Done